### PR TITLE
doc: samples: display: update documentation

### PR DESCRIPTION
**Pull request by glaubermaroto**
- fixes sample codes' relative paths
- updates building instructions.

### CODE REVIEW

File: samples/subsys/display/lvgl/README.rst [hunk 1]
```diff
@@ -24,7 +24,7 @@ for Arduino connectors, for example:
 
 or a simulated display environment in a native Posix application:
 
-- :ref:`native_posix`
+- :ref:`native_posix_64`
```
Comment (jfischer-no) [inline samples/subsys/display/lvgl/README.rst:27]: Why the change from native_posix to native_posix_64?

Comment (glaubermaroto) [reply samples/subsys/display/lvgl/README.rst]: @jfischer-no  I have mistakenly replaced it here. It is now restored to its original.

File: samples/subsys/display/lvgl/README.rst [hunk 2]
```diff
@@ -49,17 +49,20 @@ Building and Running
...
    :board: nrf52840dk_nrf52840
    :shield: adafruit_2_8_tft_touch_v2
    :goals: build flash
 
 Example building for :ref:`native_posix`:
 
-.. zephyr-app-commands::
-   :zephyr-app: samples/display/lvgl
-   :board: native_posix
-   :goals: build flash
```
Comment (jfischer-no) [inline samples/subsys/display/lvgl/README.rst:62]: I suggest to keep it, update zephyr-app, and change goals to `:goals: build run`, 
in the docu it will look like this:
```
west build -b native_posix samples/subsys/display/lvgl
west build -t run
```

Comment (glaubermaroto) [reply samples/subsys/display/lvgl/README.rst:62]: Thanks for the suggestion. Updated.

Comment (jfischer-no) [reply samples/subsys/display/lvgl/README.rst:62]: Have a few changes been lost?

**Review by dcpleung — APPROVED** [review]

**Review by jfischer-no — APPROVED** [review]

File: samples/subsys/display/lvgl/README.rst (hunk 2 above)
Comment (glaubermaroto) [reply samples/subsys/display/lvgl/README.rst:62]: Yes. For some reason, the above excerpt was not tagged as outdated after the fixing push.
